defaults/main.yml

```diff
@@ -304,6 +304,9 @@ rhel7stig_time_service_configs:
 
 rhel7stig_firewall_service: firewalld
 
+# RHEL-07-031010
+rhel7stig_log_aggregator: false
```

Comment:
how about rhel7stig_system_is_log_aggregator, similar to rhel7stig_system_is_router above?

Reply:
Sure.